source/2019-12-16-countdown-to-the-new-year-2019-ember-intl.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: Ember Test Selectors
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber
+alias: 2019/12/16/countdown-to-the-new-year-2019-ember-intl
+responsive: true
+---
+
+In our end of year series - "Countdown to The New Year: 31 Days of Ember Addons" we show case a new addon each day until the new year, and we hope you have fun hearing about our favorite addons!
+
+## Day 16
+
+For day 16 of our **Countdown to The New Year** we want to highlight and addon that truly brings the entire Ember community **from all locales together**: [Ember Intl](https://emberobserver.com/addons/ember-intl)!
+
+### What It Does
+
+Ember Intl is probably one of the most popular addons used in Ember apps today. As the go-to solution for bringing internationalisation to our application, it allows us to manage, translations, date, time and currency formats in more than 150 languages!
+
+### Why I Like It
+
+The rich Ember Intl service API and many useful template helpers help us to add new locales to our apps continually as our product's user base grows globally.
+
+For those of us who have lots of languages to support in their application, Ember Intl even allows to load translations asynchronously - making sure that the size of the bundle that determines inital page load time stays small and neat!
+
+Using Ember Intl we can build Ember apps that are accessible beyond language barriers - making it more fun to share our app to friends all around the world!
+
+[Check out Ember Intl on Ember Observer](https://emberobserver.com/addons/ember-intl)! If you are a fan of multi-locale apps and you're using internationalisation addons yourself, let us know in the comments which ones are you favorite and the most fun to use!
```

Comment:
any reason for linking to Ember Observer instead of using a link to the project directly?

Reply:
We want - if I understood correctly - refer to the Ember Observer page across all posts of the December series